### ISSUE TITLE
Allow for small, medium, large sizes of controls

### ISSUE THREAD

**Issue by roackarul** [eek/rangestepper#8]
Both the steps and the draggable marker should be customizable. Bigger controls are more user friendly, smaller controls fit better in smaller places.

**Comment by eek** [comment]
Should the small, medium, large size, also affect bullets? What do you think ?

**Comment by roackarul** [comment]
Yes, a small version would only include bullets for far left/far right positions, while the intermediate bullets/steps should be replaced by a small vertical line.

**Comment by roackarul** [comment]
Here's how this should look:
![image](https://cloud.githubusercontent.com/assets/10436242/6142180/eb97d2f8-b1b9-11e4-9765-c23d738dd9b3.png)
